src/edu/csus/ecs/pc2/core/model/Problem.java

```diff
@@ -2002,4 +2048,131 @@ public void setVivaInputValidatorHasBeenRun(boolean vivaInputValidatorHasBeenRun
         }
         vivaSettings.setVivaInputValidatorHasBeenRun(vivaInputValidatorHasBeenRun);
     }
+
+    /**
+     * Returns the currently configured memory limit (in MB) for this Problem.
+     * A memory limit of zero indicates "no limit".
+     * Note that memory limits are not enforced unless a sandbox has been selected 
+     * on the Edit Problem GUI (or via YAML configuration).
+     * 
+     * @return
+     */
+    public int getMemoryLimitMB() {
+        return memoryLimitMB;
+    }
+    
+    /**
+     * Sets the memory limit for this problem. Setting a memory limit of zero indicates "no limit",
+     * meaning that the problem is constrained only by the memory provided by the hardware, the OS,
+     * and the specific language runtime system.
+     * 
+     * Note that setting a memory limit does not automatically imply that such limit is enforced; 
+     * enforcing a memory limit requires selection of a problem sandbox capable of doing that.
+     * (See {@link EditProblemSandboxPane}.)
+     * 
+     * If a value less than zero is passed in the memory limit is set to zero (unlimited).
+     * 
+     * @param memLimitInMB the memory limit for the problem, in MB; must be >= 0, where 0=unlimited.
+     */
+    public void setMemoryLimit(int memLimitInMB) {
```

Comment:
I think this setter name should be changed to `setMemoryLimitMB()`.  This would make it clearer to future developers what was expected, would make it consistent with the corresponding getter (`getMemoryLimitMB()`), and also make it consistent with the corresponding method `setMemoryLimitInMeg()` in class `Constants`.

Reply:
Changed in [this commit](https://github.com/pc2ccs/pc2v9/pull/732/commits/ac45863596ea4b46d29a95e70db66998c8b423d5)